fastfield_codecs/src/lib.rs

```diff
@@ -19,10 +21,32 @@ pub trait FastFieldCodecReader: Sized {
     fn max_value(&self) -> u64;
 }
 
+pub trait FastFieldCodecReaderU128: Sized {
+    /// reads the metadata and returns the CodecReader
+    fn open_from_bytes(bytes: &[u8]) -> std::io::Result<Self>;
+
+    /// Get value for doc
+    fn get(&self, doc: u64, data: &[u8]) -> Option<u128>;
```

Comment:
what is `data` here? Don't we `open_from_bytes` ?

Reply:
Same as `FastFieldCodecReader.
open_from_bytes, reads the metadata from the footer, but does not use the data content